src/mame/heathkit/h89.cpp

```diff
@@ -362,9 +363,15 @@ void h89_state::machine_reset()
 
 	update_gpp(0);
 	update_mem_view();
+	m_h37->reset();
+	m_intr_cntrl->reset();
+	m_console->reset();
+	m_serial1->reset();
+	m_serial2->reset();
+	m_serial3->reset();
+	m_maincpu->reset();
```

Comment:
Once again, you shouldn’t be resetting child devices here – that happens automatically.